tests/test_requests.py

```diff
@@ -987,6 +988,15 @@ def test_response_decode_unicode(self):
         chunks = r.iter_content(decode_unicode=True)
         assert all(isinstance(chunk, str) for chunk in chunks)
 
+    def test_response_reason_unicode(self):
+        # check for unicode HTTP status
+        r = requests.Response()
+        r.url = six.u('unicode URL')
+        r.reason = six.u('Komponenttia ei löydy').encode('utf-8')
```

Comment:
Perhaps I'm missing something, why not `u'Komponenttia ei löydy'.encode('utf-8')`?